File: lab-4/P1.java
// 1. WAP to print addition of 3 numbers (with Scanner). (A)

import java.util.Scanner;

public class P1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int num1 = scanner.nextInt();
        int num2 = scanner.nextInt();
        int num3 = scanner.nextInt();

        int sum = num1 + num2 + num3;
        System.out.println("Sum: " + sum);

        scanner.close();
    }
}

File: lab-1/p1.c
// 1. Calculate average of three numbers (A)

#include <stdio.h>

void main() {
    int num1, num2, num3;

    printf("Enter three numbers:\n");
    scanf("%d %d %d", &num1, &num2, &num3);

    float average = (num1 + num2 + num3) / 3.0;

    printf("Average: %.2f\n", average);
}
